ca.go

```diff
@@ -1,11 +1,17 @@
 package main
 
 import (
+	"crypto"
 	"io"
 
+	"github.com/aws/aws-lambda-go/events"
 	"golang.org/x/crypto/ssh"
 )
 
+type SignerChooser interface {
```

Comment:
I just realized this code doesn't have `golint` running as part of CI (that is an oversight) -- would you mind adding a comment to `SignerChooser` explaining what it is in a way that will cause `golint` to be happy?